test_conformance/atomics/test_atomics.cpp

```diff
@@ -1292,8 +1292,8 @@ cl_long test_atomic_and_result_long(size_t size, cl_long *startRefValues,
 int test_atomic_and(cl_device_id deviceID, cl_context context,
                     cl_command_queue queue, int num_elements)
 {
-    TestFns set = { 0xffffffff,
-                    0xffffffffffffffffLL,
+    TestFns set = { (int)0xffffffff,
+                    (long long)0xffffffffffffffffLL,
```

Comment:
It might be better to cast these to the types used in the structure, which are guaranteed to have a fixed size, rather than `int` and `long long` that could be different sizes for different compilers.

```suggestion
    TestFns set = { (cl_int)0xffffffff,
                    (cl_long)0xffffffffffffffffLL,
```

Reply:
Done